### DISCUSSION TITLE
Writing for people who have limited or no English language

### DISCUSSION THREAD

**Discussion by Sarah-Lombardi** [UKHomeOffice/design-system#466] [Ideas]
Several of our products and services are used by people with limited or no English language. This is a problem that Home Office services face more than many other government departments so we're in a good position to lead on this.

Designers need to think about how to communicate across these barriers. Sharing tips and patterns with eachother will help.

In addition to the usual advice like using plain English and not using jargon or colloquial language, other things i've heard so far are:
* Using digital services like Notify is better as the content can be run through online translators (used in refugee integration loans)
* Using images rather than words, particularly in signage (tested in Asylum)
* Not using words or punctuation marks that could have double meaning

There was a [blog published in 2015](https://hodigital.blog.gov.uk/2015/12/29/tips-for-writing-for-non-native-english-speakers/). It would be great to update this into more formal guidance with recent examples from a number of our services.

**Comment by frankramsay** [comment]
We are trialling the use of images in digital signage at Western Jet Foil and Manston where asylum seekers and others are processed after arrival. Digital signs show a sequence of screens, so users can have limited time to perceive the content. Also, depending on their proximity to the screen, users may view the screen from a distance or at an angle. 

We believe simple images that accompany words can aid users' understanding of what is happening - especially where manual processes are involved, like showing a passport, taking a covid test or bagging possessions. Currently, this is an assumption - the effectiveness of such images for users needs to be measured.

**Comment by Sarah-Lombardi** [comment]
Research has been done on digital inclusion for eVisas. It found some issues with our content and Google Translate. Certain acronyms, like UKVI, were not translated. And where they were expanded the translation was done literally and did not make sense.

From the research:

- Most users need written and spoken English to be translated​
- Some use Google Translate​
- GT doesn’t work in all languages​
- GT is not always right and does not translate acronyms like ‘UKVI’​
- Others speaking English as a second language need help with government and eVisa content​

**Comment by Sarah-Lombardi** [comment]
Recent intake testing showed that users did not understand:
* identity
* religion
* ethnicity
* background
* dialect

They also found the future tense 'How your information will be used' harder to understand. Could easily use present tense there.

Most said they would use Google Translate.

**Comment by Sarah-Lombardi** [comment]
Final draft agreed by working group and shared with a content designer for a 2i. Then ready for publishing.
